lib/go-tc/roles.go

```diff
@@ -19,6 +28,70 @@ package tc
  * under the License.
  */
 
+// RoleV5 is an alias for RoleV50.
```

Comment:
While this is technically true, the purpose of the alias is to always point to the latest minor version for its associated major version.